SVSModel/Configuration/Constants.cs

```diff
@@ -14,7 +12,7 @@ public static class Constants
         public const double InitialN = 50;
 
         /// <summary>Dictionary containing values for the proportion of maximum DM that occurs at each predefined crop stage</summary>
-        public static readonly Dictionary<string, double> PropnMaxDM = new()
+        public static readonly Dictionary<string, double> PropNMaxDM = new()
```

Comment:
Not major but this change is not really correct.  the lower case n was part of a shotening of proportion.  Changing it to upper case infers nitrogen which is not correct for this variable.  Would be better to remove the n completly